web_src/css/user.css

```diff
@@ -26,9 +26,14 @@
 }
 
 .user.profile .ui.card .extra.content > ul > li:not(:last-child) {
+  display: flex;
   border-bottom: 1px solid var(--color-secondary);
 }
 
+.user.profile .ui.card .extra.content > ul > li:not(:last-child) > :nth-child(2) {
```

Comment:
This CSS selector seems quite tricky and fragile. I think there could be better solutions.

Reply:
I think it's not right. It would pollute some elements in 

```
				<li>
					<div class="render-content markup">{{$.RenderedDescription|Str2html}}</div>
				</li>
```

I will propose a new fix for it.